native/membrane_videocompositor_wgpu/src/lib.rs

```diff
@@ -10,6 +11,13 @@ pub struct RawVideo {
     pub pixel_format: rustler::Atom,
 }
 
+#[derive(Debug, rustler::NifStruct)]
+#[module = "Membrane.VideoCompositor.Implementations.OpenGL.Native.Rust.Position"]
+struct Position {
```

Comment:
Can't we just use `u32` and convert when needed?

Reply:
What benefit would that bring?

Reply:
I think it's just more natural to keep the position as `u32`. I have used it in OpenGL transformations. We need to do type conversion in some place anyway, but in my opinion, doing `u32` to `usize` conversion before indexing arrays improves readability and clearly explains why we need `usize`. When I first looked at the struct with `usize` type in it, I thought it comes from some rustler or WGPU/OpenGL specification.